spec/classes/init_spec.rb

```diff
@@ -238,14 +238,6 @@
         it_behaves_like 'non enabled feature'
       end
 
-      context 'accept_trusted_publisher_certs =>' do
```

Comment:
I didn't see a comment in the body of the commit - why is this test being removed? I haven't had my coffee yet but it _looks_ like a legit test?

Reply:
hey @michaeltlombardi 
I removed this test because it's duplicated by the previous test (https://github.com/puppetlabs/puppetlabs-wsus_client/blob/e544a3aecd9d2186c00f1aaf99d6d6238356d30a/spec/classes/init_spec.rb#L233)
If i'm wrong, please let me know :)

Reply:
Oh! Derrrrrp, I didn't see the test above without expanding! Good call!